### PR TITLE
Fix #353 Improve code blocks in documentation

### LINKED ISSUE #353: Improve code blocks in documentation

### PR DESCRIPTION
**Pull request by pauloxnet**

### CODE REVIEW

**Review by jezdez — APPROVED** [review]
Just a small typo

File: docs/values.rst [hunk 1]
```diff
@@ -125,7 +127,9 @@ Allow final value to be used outside the configuration context
 
 You may use the ``environ_name`` parameter to allow a :class:`~Value` to be
 directly converted to its final value for use outside of the configuration
-context::
+context:
+
+.. code-block:: pycon
```
[the remark this answers is not in the document]
Comment (pauloxnet) [reply docs/values.rst:132]: Actually `pycon` is for code block containing Python console example.

We recently used it also in the Django documentations (es: https://github.com/django/django/blob/main/docs/faq/models.txt#L13)

[the remark this answers is not in the document]
Comment (marksweb) [reply docs/values.rst:132]: Not just a case of pycon on the brain 👀😂

**Review by marksweb — APPROVED** [review]
All looks sensible to me 👍

File: docs/values.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (pauloxnet) [reply docs/values.rst:132]: See also https://pygments.org/docs/lexers/#pygments.lexers.python.PythonConsoleLexer

**Pull request merged**